### ISSUE TITLE
Log level is not logged in production mode for v0.11

### ISSUE THREAD

**Issue by pawanrawal** [balderdashy/sails#3360]
There seems to be some problem with the logging of the log level when running app in production mode. 

When I run `sails lift` output is
`info: Server lifted in '/Users/pawan/Postman/test'`

When I run `sails lift --prod` output is
`Server lifted in '/Users/pawan/Postman/test' `

In production the log-level `info` is not printed here. Any pointers to why this is so and how the behaviour can be made similar.

**Comment by shamasis** [comment]
+1

**Comment by sailsbot** [comment]
Thanks for posting, @pawanrawal.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by shamasis** [comment]
This @sailsbot is not good man. It closes tickets just because nobody chose to reply?
